app/helpers/application_helper.rb

```diff
@@ -1219,7 +1219,10 @@ def allowed_filter_db?(db)
   end
 
   def miq_tab_header(id, active = nil, options = {}, &block)
-    content_tag(:li, :class => "#{options[:class]} #{active == id ? 'active' : ''}", :id => "#{id}_tab") do
+    content_tag(:li,
+                :class     => "#{options[:class]} #{active == id ? 'active' : ''}",
+                :id        => "#{id}_tab",
+                'ng-click' => "changeAuthTab('#{id}');") do
```

Comment:
Aah, thanks :) Ok, I'll move it to that template then, so that we explicitly pass `{'ng-click' => "changeAuthTab('something')"}` in there...